app/mailers/immediate_mailer.rb

```diff
@@ -5,14 +5,17 @@
 ###
 
 class ImmediateMailer < ApplicationMailer
-  def immediate
-    to = params[:to]
-    @message = params[:message]
-
+  def immediate(message, to, delivery_method_options = nil)
```

Comment:
I might rename `to` to something else just to be safe. Feels like that could be hypothetically masking a method.